src/command.rs

```diff
@@ -81,7 +82,7 @@ impl DockerCommandBuilder {
             .args(&["--name", &self.name])
             .args(&self.build_volume_switches())
             .args(&self.build_environment_switches())
-            .args(&self.build_docker_switches())
+            .args(self.build_docker_switches())
```

Comment:
Just seems weird that the other functions here are using & but this one isn't.

Reply:
The inner function returns a reference, so there is an implicit `&`, but it does suggests there may be simplifications to be had in the other inner functions. I'll take a look.

Reply:
Ok, I think there are some further simplifications to be had here, but I'm going to address them separately.